tests/features/targets/performance.feature

```diff
@@ -1,7 +1,5 @@
 Feature: Performance and Consistence adding and removing targets
-    As an admin,
-    Admin has deployed and bootstrapped RSTUF
-    Admin has issued a Token with scopes 'write:targets', 'delete:targets' and "read:tasks"
+    User has deployed and bootstrapped RSTUF
```

Comment:
I like the two sentences you used for `add_targets.feature`, more explicit and uses the terms described in our guide as well.
```suggestion
    User has deployed RSTUF,
    User has run the ceremony and completed bootstrap successfully
```